tool/planet/agent.go

```diff
@@ -477,6 +503,47 @@ func status(c statusConfig) (ok bool, err error) {
 	return ok, nil
 }
 
+func validateKubernetesService(ctx context.Context, serviceCIDR net.IPNet) error {
+	return utils.Retry(ctx, math.MaxInt64, 5*time.Second, func() error {
```

Comment:
> math.MaxInt64

What's this? I assume you were going for an endless retry loop, so why not do it properly?